src/assets/Azure.Core.Shared/Utf8JsonWriterExtensions.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json;
+
+namespace Azure.Core
+{
+    internal static class Utf8JsonWriterExtensions
+    {
+        public static void WriteDateTimeOffsetValue(this Utf8JsonWriter writer, DateTimeOffset value, string format)
```

Comment:
Why not make this method an overload of WriteStringValue?

Reply:
Great idea. Done.